plinth/views.py

```diff
@@ -30,7 +30,20 @@
 
 from . import forms, frontpage
 import plinth
+from django.utils.translation import ugettext_lazy as _
 
+from plinth import cfg
+
+
+def apps_init():
```

Comment:
@500swapnil I think you copied instead of moving. (It should be removed from plinth/views.py.)